src/python/dxpy/bindings/apollo/dataset.py

```diff
@@ -0,0 +1,56 @@
+from dxpy import DXHTTPRequest
+
+
+class Dataset:
+    def __init__(self, record_id, project):
+        self.record_id = record_id
+        self.project = project
+
+        self.populate_http_request()
+
+    def populate_http_request(self):
```

Comment:
The fact that info is drawn from a request object is more of an internal implementation fact. I would call it by what it does: `call_visualize` or `get_visualize_info` or similar. The variable then `vizualise_info`?

Reply:
sounds good :)